packages/nouns-api/src/services/ideas.ts

```diff
@@ -379,7 +396,23 @@ class IdeasService {
       throw new Error('Idea not found for comment');
     }
 
-    return getIsClosed(idea);
+    return idea.locked || getIsClosed(idea);
+  }
+
+  // Lock ideas with the latest closingVoteCount to prevent voting history from changing
+  // if users buy/sell lil nouns
+  static async batchLockIdeas(ideas: any) {
+    for await (const idea of ideas) {
+      await prisma.idea.update({
+        where: { id: idea.id },
+        data: {
+          locked: true,
```

Comment:
why do we have to reset this lock to true? I thought these were already locked?
This is what we are doing above -- 
I'm confused how they are originally getting their locked status

```
let closed = idea.locked;
if (!closed) { // add to list that gets sent to batchLockIdeas }
```